README.md

```diff
@@ -35,7 +35,7 @@ Next in order to trigger the tests, deploy `assets/clair-config.yaml` on to your
 * `CLAIR_TEST_CONTAINERS` - String with comma separated list of conatiner images.
 * `CLAIR_TEST_RUNID`(Optional) - String specifying the desired RUNID of the test run.
 * `CLAIR_TEST_PSK` - Psk string which can be found at `~/clair/config.yaml` in the clair app pod.
-* `CLAIR_TEST_REPO_PREFIX` - String indicating the test repo prefix.
+* `CLAIR_TEST_REPO_PREFIX` - String indicating comma separated test repo prefixes. All test repo prefixes combined together shoud add up to our hitsize specified as input param.
```

Comment:
Is this true? If I have 2 test repo prefixes I need a hitsize of 2?

Reply:
Wrong wording. Rephrased the statement